app/create.js

```diff
@@ -26,7 +26,20 @@ define(['app/config', 'app/database', 'jquery', 'bootstrap', 'app/select', 'app/
 			function(callback) {
 				// First create the element
 				newElement = $('#item-layout').clone(true).appendTo("#items table").removeAttr("id");
-				callback(null, "one");
+				async.during(
```

Comment:
I have no idea what's happening here. I'll leave it up to you to figure out if that is due to async being used or your code.